app/data_sources/zendesk.py

```diff
@@ -0,0 +1,192 @@
+import logging
+from abc import abstractmethod, ABC
+from datetime import datetime
+from enum import Enum
+from typing import Dict, List, Optional
+from data_source_api.exception import InvalidDataSourceConfig
+from data_source_api.basic_document import BasicDocument, DocumentType
+from indexing_queue import IndexingQueue
+import re
+import json
+import base64
+import requests
+from parsers.html import html_to_text
+from pydantic import BaseModel
+
+from db_engine import Session
+from schemas import DataSource
+
+#import http.client as http_client
+#http_client.HTTPConnection.debuglevel = 1
+
+class HTMLInputType(Enum):
```

Comment:
why those classes are here?

Reply:
Low level debugging while I was figuring out api-calls. Mostly obsolete now but useful when I forget how to spell